build.gradle

```diff
@@ -177,8 +177,13 @@ subprojects {
             cacheChangingModulesFor 5, 'days'
         }
 
-        exclude(group: "ch.qos.logback", module: "logback-core")
-        exclude(group: "ch.qos.logback", module: "logback-classic")
+        if (!project.name.contains('cassandra-ticket-registry')) {
```

Comment:
As mentioned previously, this will break CAS basic functionality with logging, etc.